src/modules/map/components/openlayers/OpenLayersInternalLayer.vue

```diff
@@ -9,18 +9,29 @@
             :exclude-source="layerConfig.excludeSource"
         />
         <OpenLayersWMTSLayer
-            v-if="layerConfig.type === LayerTypes.WMTS"
+            v-if="layerConfig.type === LayerTypes.WMTS && !layerConfig.isExternal"
```

Comment:
As it seems that `OpenLayersWMTSLayer` component is Geoadmin specific as oposed to `OpenLayersWMSLayer` component I proposed to rename it to `OpenLayersGeoadminWMTSLayer`.

Reply:
I've added more comment to cover why external layers are now present there.